db/migrate/20230411032053_insert_community_to_sidebar_sections.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+class InsertCommunityToSidebarSections < ActiveRecord::Migration[7.0]
+  COMMUNITY_SECTION_LINKS = [
+    {
+      name: I18n.t("sidebar.sections.community.links.everything.content", default: "Everything"),
+      path: "/latest",
+      icon: "layer-group",
+      segment: SidebarUrl.segments["primary"],
+    },
+    {
+      name: I18n.t("sidebar.sections.community.links.my_posts.content", default: "My Posts"),
+      path: "/my/activity",
+      icon: "user",
+      segment: SidebarUrl.segments["primary"],
+    },
+    {
+      name: I18n.t("sidebar.sections.community.links.review.content", default: "Review"),
+      path: "/review",
+      icon: "flag",
+      segment: SidebarUrl.segments["primary"],
+    },
+    {
+      name: I18n.t("sidebar.sections.community.links.admin.content", default: "Admin"),
+      path: "/admin",
+      icon: "wrench",
+      segment: SidebarUrl.segments["primary"],
+    },
+    {
+      name: I18n.t("sidebar.sections.community.links.users.content", default: "Users"),
+      path: "/u",
+      icon: "users",
+      segment: SidebarUrl.segments["secondary"],
+    },
+    {
+      name: I18n.t("sidebar.sections.community.links.about.content", default: "About"),
+      path: "/about",
+      icon: "info-circle",
+      segment: SidebarUrl.segments["secondary"],
+    },
+    {
+      name: I18n.t("sidebar.sections.community.links.faq.content", default: "FAQ"),
+      path: "/faq",
+      icon: "question-circle",
+      segment: SidebarUrl.segments["secondary"],
+    },
+    {
+      name: I18n.t("sidebar.sections.community.links.groups.content", default: "Groups"),
+      path: "/g",
+      icon: "user-friends",
+      segment: SidebarUrl.segments["secondary"],
+    },
+    {
+      name: I18n.t("sidebar.sections.community.links.badges.content", default: "Badges"),
+      path: "/badges",
+      icon: "certificate",
+      segment: SidebarUrl.segments["secondary"],
+    },
+  ]
+  def up
+    result = DB.query <<~SQL
+      INSERT INTO sidebar_sections(user_id, title, public, section_type, created_at, updated_at)
+      VALUES (-1, 'Community', true, #{SidebarSection.section_types["community"]}, now(), now())
```

Comment:
```suggestion
      VALUES (-1, 'Community', true, 0, now(), now())
```

We need to avoid referencing the model to avoid making the migration brittle over time.